ui/article-renderer/src/main/java/org/cru/godtools/article/activity/ArticlesActivity.java

```diff
@@ -67,6 +72,17 @@ protected void onStart() {
     public void onArticleSelected(@Nullable final Article article) {
         Timber.tag("ArticlesActivity")
                 .d("Article selected: %s", article != null ? article.title : null);
+        AEMArticleItemFragment fragment = (AEMArticleItemFragment) getSupportFragmentManager()
+                .findFragmentByTag(AEMArticleItemFragment.TAG);
+
+        if (fragment == null) {
+            fragment = AEMArticleItemFragment.newInstance(mTool, mLocale, article.uri.toString());
+        } else {
+            fragment.setArticle(article);
+        }
+
+        getSupportFragmentManager().beginTransaction().replace(R.id.frame, fragment,
+                AEMArticleItemFragment.TAG).addToBackStack(AEMArticleItemFragment.TAG).commit();
```

Comment:
do this in a separate activity, don't overload the ArticlesActivity

Reply:
Should the Activity be in the article-renderer or in articles-aem-renderer

Reply:
articles-aem-renderer

Reply:
Ok